config/build/webpack.conf.js

```diff
@@ -6,6 +6,7 @@ const
 	webpack = require('webpack'),
 	StatsWriterPlugin = require('webpack-stats-plugin').StatsWriterPlugin,
 
+	clientPath = path.posix.resolve('./src/client/'),
```

Comment:
If we keep this extracted into a reused variable, please don't list it in with the other require statements.  Just put it in its own block of declared common const variables and maybe name it 'clientSrcPath' to be more descriptive